DataFormats/HcalDetId/interface/HcalDetId.h

```diff
@@ -4,6 +4,7 @@
 #include <iosfwd>
```

Comment:
this file was merged already with #23386, but the diff still shows up.
Please rebase to CMSSW_10_2_0_pre5 (or later)